### PR TITLE
Update Important note section to clarify memory allocation

### PR DESCRIPTION
**Pull request by DrShadow34**
Added example of mem allocation with 4k and changed a previous sentence to mention about uncompressed image

### CODE REVIEW

**Comment by vaxerski** [comment]
a 4K image is 33.1MB uncompressed.

File: README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ wallpaper = monitor,contain:/path/to/image.jpg
 A Wallpaper ***cannot*** be applied without preloading. The config is ***not*** reloaded dynamically.
 
 ## Important note to the inner workings
-Preload does exactly what it says. It loads the entire wallpaper into memory. This can result in around 8 - 20MB of mem usage. It is not recommended to preload every wallpaper you have, as it will be a) taking a couple seconds at the beginning to load and b) take 100s of MBs of disk and RAM usage.
+Preload does exactly what it says. It loads the entire wallpaper into memory as uncompressed. This can result in unreasonable high mem usage at higher resolutions, as an example: for 4k screen each preloaded image will allocate ~132.7MB. It is not recommended to preload every wallpaper you have, as it will be a) taking a couple seconds at the beginning to load and b) take 100s of MBs of disk and RAM usage.
```
Comment (vaxerski) [inline README.md:97]: "as uncompressed" -> "without any compression"

"unreasonable high" -> "high"

"4k screen" -> no, it depends on your _image's resolution_, not screen's.

Comment (DrShadow34) [reply README.md:97]: With first one I agree, I'm not native english speaker and trying to correct readme.md was silly from me to begin with. 
Second one - no, that unreasonable high. I got almost 2gb memory consumption from just 11 images, and they using ~140mb on disk.  
Third one - no, that make no sense. Like, as example, I preload 16000 × 10216 image, how much that would weight then if it wouldn't be converted to my screen resolution? And why would other image that 400x400 allocate same amount of memory (132.7MB per image) as previous one then?

Comment (vaxerski) [reply README.md:97]: 2 - 140MB would be like 6000x6000
3 - if it does, that's a bug.

**Pull request closed without merging**